astroplan/telescope.py

```diff
@@ -0,0 +1,139 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+						unicode_literals)
+
+# Third-party
+import numpy as np
+import astropy.units as u
+
+
+__all__ = ["Telescope"]
+
+AD_ERR_DEFAULT = np.sqrt(0.289) * (u.adu / u.pixel)
+
+
+def _default_ccd_response():
+	"""
+	A generic response function of a silicon detector from the
+	table on the following page:
+	https://www.apo.nmsu.edu/arc35m/Instruments/ARCTIC/#3p5
+	"""
+	wl = [100., 309.45600215, 319.53901519, 329.99173871,
+			  339.70504127, 349.78805431, 379.53294279, 390.00376402,
```

Comment:
Inconsistent indentation. If you are mixing tabs and spaces, replace tab with 4 spaces.